optimism/src/keccak/witness.rs

```diff
@@ -15,6 +15,28 @@ use super::{
     DIM, HASH_BYTELENGTH, QUARTERS, WORDS_IN_HASH,
 };
 
+pub(crate) fn pad_blocks<Fp: Field>(pad_bytelength: usize) -> Vec<Fp> {
```

Comment:
Follow-up: I would prefer to have a fixed-length array instead of a vector as we know the size. Better for performances as it is known at compile time.